models/models.go

```diff
@@ -75,6 +75,32 @@ type Dimension struct {
 	Options []string `bson:"options"                 json:"options"`
 }
 
+// PublicDimension represents information about a single dimension as served by /dimensions and /dimensions/<id>
+type PublicDimension struct {
+	Name  string                  `bson:"name"                    json:"name"`
+	Links *PublicDimensionLinkMap `bson:"links"                    json:"links"`
+}
+
+// PublicDimensionLinkMap is the links map for the PublicDimension structure
+type PublicDimensionLinkMap struct {
+	Self    LinkObject `bson:"self"                 json:"self"`
+	Filter  LinkObject `bson:"filter"               json:"filter"`
+	Options LinkObject `bson:"options"              json:"options, omitempty"`
+}
+
+// PublicDimensionOptions represents information about a single dimension option as served by /options and /options/<id>
+type PublicDimensionOption struct {
+	Links  *PublicDimensionOptionLinkMap `bson:"links"                    json:"links"`
+	Option string                        `bson:"option"              json:"option"`
+}
+
+// PublicDimensionOptionLinkMap is the links map for the PublicDimensionOption structure
+type PublicDimensionOptionLinkMap struct {
+	Self      LinkObject `bson:"self"                 json:"self"`
+	Filter    LinkObject `bson:"filter"               json:"filter"`
+	Dimension LinkObject `bson:"dimension"               json:"dimension"`
+}
```

Comment:
in all of the above, the alignment in the `json:...` part of the tags could be better 👀 
*twitch*